recipes/forestci/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "forestci" %}
+{% set version = "0.1" %}
+{% set sha256 = "e706ec046c802c2e1036fd8ee306f1d9841b1a9239d0346f8c1849c5966578e0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - scikit-learn
+    - numpy
+    - nose
```

Comment:
`nose` appears only to be needed for testing, but shouldn't be needed to run the package, so I would remove this.